src/pattern-library/components/patterns/SharedFormPatterns.js

```diff
@@ -0,0 +1,52 @@
+import { Checkbox } from '../../../';
+
+import {
+  PatternPage,
+  Pattern,
+  PatternExamples,
+  PatternExample,
+} from '../PatternPage';
+
+function gimmeSandwich(isChecked) {
```

Comment:
That's a bit unusual. Is there a reason why you don use `useState` inside the `SharedFormComponent` component?